test/vanilla/AcceptanceTests/acceptanceTests.ts

```diff
@@ -1806,18 +1806,6 @@ describe('typescript', function () {
 
     describe('Files Client', function () {
       var testClient = new AutoRestSwaggerBATFileService(baseUri, clientOptions);
-      it('should correctly deserialize binary streams', function (done) {
```

Comment:
why was this test removed?

Reply:
i removed it because it does not work with LF eols only with cr lf and i had issues, but i never wanted to push those changes 💃